package.json

```
@@ -25,40 +25,42 @@
   "author": "Felix Heidecke",
   "license": "AGPL-3.0",
   "dependencies": {
-    "axios": "0.21.1",
-    "babel-cli": "6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-loader": "6.4.1",
-    "babel-polyfill": "6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "css-loader": "1.0.0",
+    "@babel/core": "^7.15.5",
+    "@babel/preset-env": "^7.15.4",
+    "@babel/runtime": "^7.15.4",
+    "acorn": "^8.4.1",
+    "acorn-import-assertions": "^1.7.6",
+    "axios": "^0.21.1",
+    "babel-loader": "^8.2.2",
+    "core-js": "^3.17.2",
+    "css-loader": "^6.2.0",
     "easygettext": "^2.17.0",
-    "expose-loader": "0.7.5",
-    "jquery": "3.6.0",
-    "less": "2.7.3",
-    "less-loader": "4.0.5",
-    "node-sass": "^4.14.1",
-    "npm-watch": "0.11.0",
-    "pug": "3.0.1",
-    "sass-loader": "6.0.7",
+    "expose-loader": "^3.0.0",
+    "jquery": "^3.6.0",
+    "less": "^4.1.1",
+    "less-loader": "^10.0.1",
+    "node-sass": "^6.0.1",
+    "npm-watch": "^0.11.0",
+    "pug": "^3.0.1",
+    "sass": "^1.39.0",
```

Pin sass to version `1.32.12` to suppress the math related warnings in styles.
```suggestion
    "sass": "1.32.12"
```
(run npm install again afterwards, so that the package-lock.json file gets updated accordingly)